osbrain/agent.py

```diff
@@ -933,9 +944,11 @@ def _handle_async_requests(self, data):
         else:
             handler(self, response)
 
-    def _subscribe(self, alias: str, handlers: Dict[Union[bytes, str], Any]):
+    def subscribe(self, alias: str,
+                  handlers: Dict[Union[bytes, str], Any]) -> None:
```

Comment:
I think we should use `handler` (singular) to be consistent with `bind`/`connect`. What you think?

Reply:
I agree. Changed all occurrences!